pkg/cmd/server/start/start_master.go

```diff
@@ -454,12 +454,14 @@ func (m *Master) Start() error {
 			openshiftConfig.Informers.Start(utilwait.NeverStop)
 			openshiftConfig.Informers.StartCore(utilwait.NeverStop)
 			openshiftConfig.AuthorizationInformers.Start(utilwait.NeverStop)
+			openshiftConfig.TemplateInformers.Start(utilwait.NeverStop)
```

Comment:
> I think @jim-minter wanted to make sure the caches were primed before any api requests were allowed

We've done that at point of use before.  Not everything needs to wait for a full sync and certainly not our apiserver as a whole.